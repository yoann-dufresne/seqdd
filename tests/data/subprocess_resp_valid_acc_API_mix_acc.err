  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   333  100   333    0     0   3490      0 --:--:-- --:--:-- --:--:--  3505
