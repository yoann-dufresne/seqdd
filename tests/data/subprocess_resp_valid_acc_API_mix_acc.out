<?xml version="1.0" encoding="UTF-8"?>
<ErrorDetails>
  <timestamp>1732889329181</timestamp>
  <status>400</status>
  <error>Bad Request</error>
  <message>All accessions must be of the same data type as the first accession, which was ASSEMBLY.</message>
  <path>/ena/browser/api/xml/GCA_003543015.1,SRS000006</path>
</ErrorDetails>
